Ginger/GingerPluginCore/CommLib/GingerSocketInfo.cs

```diff
@@ -241,6 +242,11 @@ private void ReceiveCallback(IAsyncResult ar)
                 }
                 throw e;
             }
+            catch(OutOfMemoryException ex)
+            {
+                Console.WriteLine("OutOfMemoryException: ", ex.Message);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The format string might be wrong, the following arguments are unused: 'ex.Message'.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5402066)